client/src/components/Loading/Loading.scss

```diff
@@ -0,0 +1,111 @@
+$dash: 750;
```

Comment:
I've moved this bit from `_style.scss` so it lives next to the actual component its meant to style.